roles/confluent.kafka_connect/defaults/main.yml

```diff
@@ -17,49 +17,8 @@ kafka_connect_service_environment_overrides:
   KAFKA_HEAP_OPTS: "-Xms256M -Xmx2G"
   KAFKA_OPTS: "{{ kafka_connect_final_java_args | java_arg_build_out }}"
 
-kafka_connect_default_internal_replication_factor: "{{ [ groups['kafka_broker'] | length, 3 ] | min }}"
-
-kafka_connect_secret_registry_enabled: "{{rbac_enabled}}"
-kafka_connect_secret_registry_key: 39ff95832750c0090d84ddf5344583832efe91ef
-
-kafka_connect_group_id: connect-cluster
```

Comment:
Might be worth leaving a comment of where the default values now live, the dictionary name etc.